### PR TITLE
[make:twig-extension] Change folder for Twig Extension

### PR DESCRIPTION
**Pull request by seb-jean**
I think it would be interesting to put the Twig Extensions in a `src/Twig/Extension` folder rather than `src/Twig`. The idea is to do like the Twig Components.

### CODE REVIEW

File: src/Maker/MakeTwigExtension.php [hunk 1]
```diff
@@ -51,7 +51,7 @@ public function generate(InputInterface $input, ConsoleStyle $io, Generator $gen
     {
         $extensionClassNameDetails = $generator->createClassNameDetails(
             $input->getArgument('name'),
-            'Twig\\',
+            'Twig\\Extension',
```
Comment (OskarStark) [inline src/Maker/MakeTwigExtension.php:54]:
```suggestion
            'Twig\\Extension\\',
```
?

**Review by jrushlow — APPROVED** [review]
Thanks @seb-jean!

**Comment by weaverryan** [comment]
Devil's advocate: what else do you have in the `src/Twig/` directory? Same thing with forms - I like putting them in `src/Form` instead of `src/Form/Type` because, most of the time, they will be the only thing in `src/Form` :)

**Comment by seb-jean** [comment]
> Devil's advocate: what else do you have in the `src/Twig/` directory? Same thing with forms - I like putting them in `src/Form` instead of `src/Form/Type` because, most of the time, they will be the only thing in `src/Form` :)

Now, I have a folder `src/Twig/Components` and files `src/Twig/...Extension.php`

**Comment by weaverryan** [comment]
haha, that's a good point 😆 . Ok then 👍

**Pull request merged**